src/__tests__/List/List.test.js

```diff
@@ -2,11 +2,11 @@ import React from 'react'
 import { connect } from 'react-redux'
```

Comment:
Instead of `List.test.js`, let's call this file `render.test.js`, since it tests the `render` method of the `List` component.